### ISSUE TITLE
LiveView — bump dependency version to 1.13.0

### ISSUE THREAD

**Issue by darraghenright** [msaraiva/surface#86]
Hi Marlus!

Someone reported [this issue](https://elixirforum.com/t/newly-created-app-hangs-on-home-page-with-error-in-phoenix-liveview-channel-start-link-1/31738) when running LiveView `1.12.1` on Elixir Forum a few days ago. 

The suggested fix is to bump Phoenix and LiveView to their latest versions.

The latest version of LiveView is  `1.13.0` but the Surface `master` branch is still pegged to `1.12.1` so it won't compile. I tested bumping version locally — this appeared to fix the issue and everything appears to be working as expected now.

That said, I forked the library and bumped the LV version, but running `mix test` failed with a compilation error:

```
== Compilation error in file lib/surface/components/live_patch.ex ==
** (RuntimeError) could not load template_engines configuration for Phoenix. Please ensure you have listed :phoenix under :applications in your mix.exs file and have enabled the :phoenix compiler under :compilers, for example: [:phoenix] ++ Mix.compilers
    lib/phoenix/template.ex:253: Phoenix.Template.raw_config/1
    lib/phoenix/template.ex:243: Phoenix.Template.compiled_engines/0
    lib/phoenix/template.ex:314: Phoenix.Template.find_all/2
    expanding macro: Phoenix.LiveView.Renderer.__before_compile__/1
    lib/surface/components/live_patch.ex:1: Surface.Components.LivePatch (module)
```

Following the above instructions I added:

```
def project do
  [
    # etc.
    compilers: [:phoenix] ++ Mix.compilers()
  ]
end
```

And the tests run now (although there are some deprecation messages).

However ElixirLS is reporting an error:

```
{
	"resource": "/private/tmp/surface/mix.exs",
	"owner": "_generated_diagnostic_collection_name_#1",
	"severity": 8,
	"message": "an exception was raised:\n    ** (Mix.Error) Can't continue due to errors on dependencies\n        (mix 1.10.3) lib/mix.ex:392: Mix.raise/1\n        (mix 1.10.3) lib/mix/tasks/deps.loadpaths.ex:28: Mix.Tasks.Deps.Loadpaths.run/1\n        (mix 1.10.3) lib/mix/task.ex:330: Mix.Task.run_task/3\n        (mix 1.10.3) lib/mix/tasks/loadpaths.ex:35: Mix.Tasks.Loadpaths.run/1\n        (mix 1.10.3) lib/mix/task.ex:330: Mix.Task.run_task/3\n        (mix 1.10.3) lib/mix/tasks/compile.ex:90: Mix.Tasks.Compile.run/1\n        (mix 1.10.3) lib/mix/task.ex:330: Mix.Task.run_task/3\n        (language_server 0.4.0) lib/language_server/build.ex:193: ElixirLS.LanguageServer.Build.compile/0",
	"source": "ElixirLS",
	"startLineNumber": 1,
	"startColumn": 1,
	"endLineNumber": 59,
	"endColumn": 1
}
```

I was going to open a PR bumping the version but I'm not sure what this is about, so I figured I'd report it.

Thanks!

**Comment by msaraiva** [comment]
Fixed in https://github.com/msaraiva/surface/commit/b878ef909ed5f08c1fda7028b3c01e1d0c00b3a9.

@darraghenright the update went just fine. I also fixed the warnings.

Regarding `ElixirLS`, I removed `.elixir_ls`, reopened the editor and waited until it compiled the project. No errors after that. Not sure if there's a better way but that's what I do whenever ElxirLS start complaining after upgrading some of the libs.

Cheers.
